pkg/config/config.go

```diff
@@ -261,11 +262,38 @@ type Config struct {
 	// IstioSidecarProxyDefaultMemory is the default Memory resource request of the istio sidecar proxy (default: 200Mi)
 	IstioSidecarProxyDefaultMemory string `yaml:"IstioSidecarProxyDefaultMemory"`
 
+	// serviceGroups defines a list of service category names.
+	ServiceGroups []ServiceGroup `yaml:"ServiceGroups"`
+	// MaximumMaxReplicasPerService is the maximum maxReplicas that tortoise can give to the HPA per service category.
+	// If the service category is not found in this list, tortoise uses the default value which is the value set in MaximumMaxReplicas.
+	MaximumMaxReplicasPerService []MaximumMaxReplicasPerGroup `yaml:"MaximumMaxReplicasPerService"`
+
 	// FeatureFlags is the list of feature flags (default: empty = all alpha features are disabled)
 	// See the list of feature flags in features.go
 	FeatureFlags []features.FeatureFlag `yaml:"FeatureFlags"`
 }
 
+type MaximumMaxReplicasPerGroup struct {
+	// ServiceGroupName refers to one ServiceGroup at Config.ServiceGroups
+	ServiceGroupName *string `yaml:"ServiceGroupName"`
```

Comment:
Now it's mandatory field, so can we:

```suggestion
	ServiceGroupName string `yaml:"ServiceGroupName"`
```

Reply:
Changed